src/functions/module_lookup.ts

```diff
@@ -27,7 +27,7 @@ const DEFAULT_MATCH_MODE = 0;
 const DEFAULT_SEARCH_MODE = 1;
 const DEFAULT_ABSOLUTE_RELATIVE_MODE = 1;
 
-function assertAvailable(variable, searchKey) {
```

Comment:
nitpicl: might as well type `variable` while you're at it. And maybe a variable name "variable" doesn't have the bestest of names 🙃

Reply:
Yes I flemmed out on this because it takes 3-4 different kinds of arguments. Seeing its use, i didn't bother too much. The typing is still wobbly atm

Reply:
>  variable: CellValue | CellValue[] | Matrix<ValueAndFormat> | ValueAndFormat[] | ValueAndFormat | undefined

So yeah ...